RockPaperScissors/RockPaperScissors/RPSGame.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  RPSGame.swift
+//  RockPaperScissors
+//
+//  Created by 전성수 on 2023/09/06.
+//
+
+import Foundation
+
+class RPSGame {
+    
+    func run() {
+        while true {
+            printMenu()
+            
+            let userChoice = getUserChoice()
```

Comment:
```suggestion
            guard let userChoice = getUserChoice() else {
                print("잘못된 입력입니다. 다시 시도해주세요.")
                continue
            }
```
=> 이런식으로요. 이렇게 만들어놓으면 게임이 어떻게 진행되고 있는지도 한눈에 더 알아볼수 있어서 좋을 것 같습니다.